src/woo_publications/logging/serializing.py

```diff
@@ -17,6 +18,8 @@ def model_to_dict(instance):
         match f:
             case models.ManyToManyField():
                 value = [obj.pk for obj in value]
+            case models.ImageField():
+                value = base64.b64encode(value.file.read()).decode("ascii")
```

Comment:
I don't know if this is a good way to solve this, but I thought this with maybe a new template to display the image properly would do the trick? If you agree then I will look into the template.